webapp/sources/rudder/rudder-web/src/main/scala/com/normation/rudder/web/services/DisplayNode.scala

```diff
@@ -512,103 +508,99 @@ object DisplayNode extends Loggable {
+      }
+      <div class="security-info">
                 {
-            agent.securityToken match {
-              case _: PublicKey   => NodeSeq.Empty
-              case c: Certificate =>
-                c.cert.either.runNow match {
-                  case Left(e)     => <span title={e.fullMsg}>Error while reading certificate information</span>
-                  case Right(cert) => (
-                    <div><label>Fingerprint (sha1): </label> <samp>{
-                      SHA1.hash(cert.getEncoded).grouped(2).mkString(":")
-                    }</samp></div>
+        agent.securityToken match {
+          case _: PublicKey   => NodeSeq.Empty
+          case c: Certificate =>
+            c.cert.either.runNow match {
+              case Left(e)     => <span title={e.fullMsg}>Error while reading certificate information</span>
+              case Right(cert) => (
+                <div><label>Fingerprint (sha1): </label> <samp>{
+                  SHA1.hash(cert.getEncoded).grouped(2).mkString(":")
+                }</samp></div>
                         <div><label>Expiration date: </label> {
-                      DateFormaterService.getDisplayDate(new DateTime(cert.getNotAfter))
-                    }</div>
-                  )
-                }
+                  DateFormaterService.getDisplayDate(new DateTime(cert.getNotAfter))
+                }</div>
+              )
             }
-          }
+        }
+      }
                 {curlHash}
                 {cfKeyHash}
                 <button type="button" class="toggle-security-info btn btn-default" onclick={
-            s"$$('#${publicKeyId}').toggle(300); $$(this).toggleClass('opened'); return false;"
-          }> <b>{tokenKind}</b> {checked}</button>
+        s"$$('#${publicKeyId}').toggle(300); $$(this).toggleClass('opened'); return false;"
+      }> <b>{tokenKind}</b> {checked}</button>
                 <pre id={publicKeyId} class="display-keys" style="display:none;"><div>{agent.securityToken.key}</div></pre>{
-            Script(OnLoad(JsRaw(s"""initBsTooltips();""")))
-          }
-              </div>
-        case None        => NodeSeq.Empty
+        Script(OnLoad(JsRaw(s"""initBsTooltips();""")))
       }
+              </div>
     }
+
         </div>
 
         <div class="status-info col-lg-6 col-sm-5 col-xs-12">
           <h3>Status information</h3>
           <div class="node-compliance-bar"></div>
           <div>
             <label>Inventory created (node local time):</label>  {
-      sm.node.inventoryDate.map(DateFormaterService.getDisplayDate(_)).getOrElse("Unknown")
+      nodeFact.lastInventoryDate.map(DateFormaterService.getDisplayDate(_)).getOrElse("Unknown")
     }
           </div>
           <div>
             <label>Inventory received:</label>  {
-      sm.node.receiveDate.map(DateFormaterService.getDisplayDate(_)).getOrElse("Unknown")
+      DateFormaterService.getDisplayDate(nodeFact.factProcessedDate)
```

Comment:
the intent is clearer now: it's not the date when the inventory was received on rudder server, but the date when it was processed by rudder web app.